cache/test_cache_control.py

```diff
@@ -22,11 +22,8 @@ class TestCacheControl(tester.TempestaTest):
     tempesta_template = {
         'config' :
             """
-            cache 2;
-            srv_group default {
-                server ${general_ip}:8000;
-            }
-            vhost vh1 {
+            server ${server_ip}:8000;
+            vhost default {
```

Comment:
What's the point for the configuration change?

Reply:
That's the way the default server group is specified in many tests, including the recent ones, so I wanted it to be more consistent.

Reply:
There is no need to keep all the tests consistent. Rather we need to test or at least to keep examples of different accepted configurations. I's better to test a SUT with as _diverse_ cases as possible